.env.example

```diff
@@ -1,16 +1,23 @@
+# API Keys
 INFURA_API_KEY=0000000000000000000000000000000000000000
-PROTOCOL_DEPLOYER_PRIVATE_KEY=0000000000000000000000000000000000000000
-CC_TOKEN_DEPLOYER_PRIVATE_KEY=0000000000000000000000000000000000000000
 ETHERSCAN_API_KEY=000000000000000000000000000000000000000
 CMC_API_KEY=000000000000000000000000000000000000000
 
-//local hardhat address #9
+# Wallet keys
+PROTOCOL_DEPLOYER_PRIVATE_KEY=0000000000000000000000000000000000000000
+CC_TOKEN_DEPLOYER_PRIVATE_KEY=0000000000000000000000000000000000000000
+
+# local hardhat address #9
 BOSON_TOKEN=0x056b1ce5510ceefa6c33600551e55333aff8296f
 
-//DAI token Rinkeby address
+# DAI token Rinkeby address
 DAI_TOKEN=0x6A9865aDE2B6207dAAC49f8bCba9705dEB0B0e6D
 
-//metadata URIs
+# metadata URIs
 CONDITIONAL_COMMIT_TOKEN_METADATA_URI=changeme
-VOUCHERSETS_METADATA_URI=changeme
-VOUCHERS_METADATA_URI=changeme
+METADATA_BASE=changeme
```

Comment:
These still aren't correct. It should be 
OUCHERSETS_METADATA_URI=changeme
VOUCHERS_METADATA_URI=changme

METADATA_BASE should be removed

Reply:
Done.